README.md

```diff
@@ -139,5 +169,4 @@ after(() => {
 ```
 
 ## Example Project
-Check out a simple react app example project at [/example/todo-example](/example/todo-example/)
-
+Check out simple react app example projects at [/example/todo-example](/example/todo-example/). Example contains examples for Cypress 10.x and Cypress 9.x.
```

Comment:
think it would be better to split out the links as the 10.x project is under a different subfolder, `example-cypress-10` instead of `example`

Check out simple react app example projects 

- Cypress 10.x [/example-cypress-10/todo-example/](/example-cypress-10/todo-example/)
- Cypress 9.x [/example/todo-example](/example/todo-example/)

or just to have one example folder, that way you could have `<example_name>-<web_tech>-<cypress_version>` if we were ever to build examples without React

`/example/todo-react-cypress-9`
`/example/todo-react-cypress-10`

Reply:
it might also be nice to list commands to run then

- npm install
- npm run cypress:open in one terminal
- npm run start in a second terminal 
- run test in cypress
- check pacts folder